presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewCandidateExtractor.java

```diff
@@ -36,174 +37,132 @@
 {
     private static final Session SESSION = testSessionBuilder(new SessionPropertyManager(new SystemSessionProperties())).build();
     private static final SqlParser SQL_PARSER = new SqlParser();
+    private static final Metadata METADATA = new MockMetadata();
```

Comment:
Mocking may be fine, but can we check if we can rely on the alter table mechanism for this implementation?

Reply:
That sounds good. But since this test is for `MaterializedViewCandidateExtractor`, how about let's just focus on the extractor's own functionality. If we want to test whether `Metadata` can retrieve the information from an altered table (with the new added parameter), maybe I can write another test to cover that (probably the `TestHiveLogicalPlanner` has already covered that part).

Reply:
Leave this part as it is as we discussed.